tests/unit/spec/pocket/updates-signup.js

```diff
@@ -0,0 +1,266 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import UpdatesForm from '../../../../media/js/pocket/updates-signup.es6';
+import sinon from 'sinon';
+
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/**
+ * Note: There is a `novalidate` attribute added to the <form>
+ * element below for testing purposes only.
+ */
+
+describe('updates-signup.es6.js', function () {
+    beforeEach(async function () {
+        const pocketUpdatesNode = `<div class="pocket-updates">
+        <section class="mzp-c-split mzp-l-split-center-on-sm-md">
+
+            <div class="mzp-c-split-container">
+
+                <div class="mzp-c-split-body">
+
+                    <div id="pocket-updates-thanks" class="pocket-updates-success hidden">
+                        <h1>Done! Keep an eye out for messages from us.</h1>
+                        <p>Until then, why not read something interesting?</p>
+                        <a class="mzp-c-button" href="/{LANG}/explore">
+                        Discover must-read articles
+                        </a>
+                    </div>
+
+                    <div id="pocket-updates-form-wrapper">
+                        <h1>Keep up with Pocket</h1>
+                        <p>Join our mailing list to stay on top of improvements, tips and product updates.</p>
+                        <form action="/en/_newsletter_subscribe/" class="mzp-c-newsletter-form pocket-updates-form" id="updates-form" novalidate>
+
+                            <div class="mzp-c-form-errors hide-from-legacy-ie hidden" id="subscribe-errors">
+                                <ul class="mzp-u-list-styled">
+                                <li class="error-email-invalid hidden">
+                                    Please enter a valid email address.
+                                </li>
+                                <li class="error-try-again-later hidden">
+                                    We are sorry, but there was a problem with our system. Please try again later.
+                                </li>
+                                </ul>
+                            </div>
+
+                            <fieldset>
+                                <label for="email" class="visually-hidden">
+                                pocket-udpates-your-email-address
+                                </label>
+                                <input type="email" name="email" id="email" placeholder="Your email address" required aria-required="true">
```

Comment:
I think we're ok just to use `required` today
```suggestion
                                <input type="email" name="email" id="email" placeholder="Your email address" required>
```